spec/index.bs

```diff
@@ -4878,12 +4878,25 @@ GPURenderPassEncoder includes GPUProgrammablePassEncoder;
 GPURenderPassEncoder includes GPURenderEncoderBase;
 </script>
 
-  * {{GPURenderEncoderBase/setIndexBuffer()}}/{{GPURenderEncoderBase/setVertexBuffer()}}:
-      * If `size` is zero, the remaining size (after `offset`) of the {{GPUBuffer}} is used.
-
   * In indirect draw calls, the base instance field (inside the indirect
     buffer data) must be set to zero.
 
+{{GPURenderEncoderBase}} has the following internal slots:
+
+<dl dfn-type=attribute dfn-for="GPURenderEncoderBase">
+    : <dfn>\[[pipeline]]</dfn>, of type {{GPURenderPipeline}}
+    ::
+        The current {{GPURenderPipeline}}, initially `null`.
+
+    : <dfn>\[[index_buffer]]</dfn>, of type {{GPUBuffer}}
+    ::
+        The current buffer to read index data from, initially `null`.
+
+    : <dfn>\[[vertex_buffers]]</dfn>, of type sequence<{{GPUBuffer}}>
+    ::
+        The current {{GPUBuffer}}s to read vertex data from, initially an empty [=/list=].
```

Comment:
nit: I think this should be a map rather than a list (it maps from slots to buffers)